### PR TITLE
Add bazel build/test jobs for 1.11 release branch

### PR DESCRIPTION
**Pull request by dims**
Looks like we are missing a few jobs.

### CODE REVIEW

**Comment by dims** [comment]
cc @timothysc

File: prow/config.yaml [hunk 1]
```diff
@@ -7507,7 +7507,35 @@ postsubmits:
...
+    agent: kubernetes
+    branches:
+    - release-1.11
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180511-51cf02e75-1.10
```
Comment (BenTheElder) [inline prow/config.yaml]: We also need to come back and fork this image to have a 1.11 variant, I'll do that later.

**Comment by BenTheElder** [comment]
testgrid config is not quite right

**Comment by luxas** [comment]
Thanks @dims, I think and hope this will unblock our kubeadm jobs which are currently failing due to missing artifacts :clap:!

**Comment by dims** [comment]
@BenTheElder y, trial-and-error-ing :)

**Pull request merged**